lib/commands/sdk.js

```diff
@@ -1444,18 +1444,21 @@ exports.getReleases = async function getReleases(config, os) {
 		// Paginate through the releases on the repo, and determine which of the assets
 		// is correct for the platform we're running on
 		const releaseData = await octokit.paginate(paginateData);
-		const osRegex = /mobilesdk-\d+\.\d+\.\d+\.GA-(\w+)\.zip/;
+		const osRegex = /mobilesdk-\d+\.\d+\.\d+\.([A-Z)]{2})-(\w+)\.zip/;
 		for (let { assets, name } of releaseData) {
 			const release = assets.find(asset => {
-				const [ , assetOS ] = osRegex.exec(asset.name);
+				const [ , versionType, assetOS ] = osRegex.exec(asset.name);
 				if (assetOS === os) {
 					return asset.url;
 				}
 				return false;
 			});
 
 			if (release) {
-				name = name.includes('GA') ? name : `${name}.GA`;
+				const analyseVersionRegExp = /(\d+\.\d+\.\d+)(\.)*([A-Z)]{2})*/;
+				const [ , version, dot, versionType ] = analyseVersionRegExp.exec(name);
+				// Fallback in GA version for backward compatibility
+				name = !versionType || !dot ? `${version}.GA` : `${version}.${versionType}`;
```

Comment:
```suggestion
				if (!version || !versionType || !dot) {
					name = name.includes('GA') ? name : `${name}.GA`;
				} else {
					name = `${version}.${versionType}`;
				}
```

Why not let as fallback the old code propose?

Reply:
This is just an idea because RC is always o the name (at least until now).